doc/leak_tracking/DETECT.md

```diff
@@ -5,12 +5,42 @@ The text below is under construction.
 
 # Detect Memory Leaks
 
-This page describes how to auto-detect certain types of memory leaks in Dart and Flutter applications and tests.
-See other information on memory leaks [here](../README.md).
+This page describes how to auto-detect certain types of memory leaks.
+Read more about leak tracking in [overview](OVERVIEW.md).
 
 ## Quick start to track leaks for Flutter
 
-### Flutter application
+### Test cover with `testWidgets`
+
+To make `testWidgets` detecting leaks in your tests, enable leak tracking in [`test/flutter_test_config.dart`](see hhttps://api.flutter.dev/flutter/flutter_test/flutter_test-library.html):
```

Comment:
The Flutter test method `testWidgets` can be configured to track and detect leaks from all instrumented classes. To enable leak tracking for your entire test suite and to configure leak tracking settings, modify your `flutter_test_config.dart` file: